HLTrigger/Muon/src/HLTMuonDimuonL3Filter.cc

```diff
@@ -161,19 +165,63 @@ HLTMuonDimuonL3Filter::hltFilter(edm::Event& iEvent, const edm::EventSetup& iSet
    Handle<RecoChargedCandidateCollection> mucands;
    if (saveTags()) filterproduct.addCollectionTag(candTag_);
    iEvent.getByToken(candToken_,mucands);
+
+   // Test to see if we can use L3MuonTrajectorySeeds:
+   if (mucands->size()<1) return false;
+   auto tk = (*mucands)[0].track();
```

Comment:
This is a tracker track; so maybe I can just dereference the track here:
https://github.com/BenjaminRS/cmssw/blob/9a5a3d38ba2cfe1bda5fb5598b46f39f6023d8d5/HLTrigger/Muon/src/HLTMuonDimuonL3Filter.cc#L175